### PR TITLE
fix!: fundInfo with updated polkadot/api

### PR DESCRIPTION
**Pull request by TarikGul**
Before publishing the next release this PR updates the fields `firstSlot` and `lastSlot` within type `FundInfo` to correctly match the type generated from `@polkadot/api`.

I updated the docs, tests, and service, as well as updated the dependencies.

### CODE REVIEW

**Review by insipx — APPROVED** [review]

File: docs/src/openapi-v1.yaml [hunk 1]
```diff
@@ -2386,10 +2386,10 @@ components:
           enum:
           - preEnding
           - ending
-        firstSlot:
+        firstPeriod:
```
Comment (niklasad1) [inline docs/src/openapi-v1.yaml:2389]: I don't understand this, the variable is just renamed and the type is not changed?

Comment (TarikGul) [reply docs/src/openapi-v1.yaml:2389]: Yea so the fields `firstSlot`, and `lastSlot` within the `FundInfo` type has been renamed to `firstPeriod` and `lastPeriod`. I worded the description wrong it should say: 

"updates the fields 'firstSlot' and 'lastSlot' within the type `FundInfo`"

**Review by niklasad1 — APPROVED** [review]

File: docs/src/openapi-v1.yaml (hunk 1 above)
Comment (dvdplm) [reply docs/src/openapi-v1.yaml:2389]: Given `FundInfo` is a public type, changing it's member names like this is a breaking change right? So the next release is major?

**Review by dvdplm — APPROVED** [review]

File: docs/src/openapi-v1.yaml (hunk 1 above)
Comment (TarikGul) [reply docs/src/openapi-v1.yaml:2389]: Yes indeed. This would be a breaking change, fixed the commit name.

**Pull request merged**